code/object/objectdock.cpp

```diff
@@ -337,6 +340,65 @@ float dock_calc_docked_speed(object *objp)
 	return dfi.maintained_variables.float_value;
 }
 
+void dock_calc_total_moi(matrix* dest, object* objp, vec3d *center)
+{
+	Assert(dest != NULL);
```

Comment:
nullptr!!

Reply:
Using NULL is common in this file, should I change all of them?

Reply:
My way to do it is to write new ones as nullptr, but leave old ones for a cleanup or refactor PR.